OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -291,6 +291,7 @@ public void onException(Exception ex) {
                         }
                     }
                     if (fieldErrorShown) {
+                        showErrorPopup();
```

Comment:
I think this is causing the alert to pop up twice.

or maybe somewhere else is not supposed to be calling the alert.